dev/com.ibm.ws.jpa.tests.eclipselink.query_31_fat.common/test-applications/jpa31/src/com/ibm/ws/jpa/jpa31/testlogic/TestMathLogic.java

```diff
@@ -17,6 +17,8 @@
 import java.util.List;
 import java.util.Map;
 
+import javax.management.Query;
```

Comment:
The code is failing to compile... the test requires `javax.persistence.Query`.... replacing it with `javax.management.Query` will not work, they are totally different things.  Need to remove `javax.management.Query` and restore the original `javax.persistence.Query`